blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAnnotationsEndToEndTest.java

```diff
@@ -153,7 +153,12 @@ private URI getAnnotationsQueryURI() throws URISyntaxException {
     @AfterClass
     public static void tearDownClass() throws Exception{
         Configuration.getInstance().setProperty(CoreConfig.EVENTS_MODULES.name(), "");
-        esSetup.terminate();
-        httpQueryService.stopService();
+        if (esSetup != null) {
```

Comment:
I added the null check for these tears down methods because it was masking the real error when something failed during the test setups and these clients never got instantiate and configured, and when it failed and called the tear down it then tries to call the terminate or stopService methods etc which resulted in a NullPointerException, and the error that was getting as the cause of the test failures.